modules/server/src/main/scala/seqexec/server/gnirs/GnirsControllerEpics.scala

```diff
@@ -201,7 +201,7 @@ object GnirsControllerEpics extends GnirsEncoders {
         c:    Camera,
         slit: Option[SlitWidth]
       ): F[Option[F[Unit]]] = {
-        lazy val specialConversion: String = s"${cameraStr(c)}+MIR${disperserStr(s.disperser)}"
+        lazy val specialConversion: String = s"${cameraStr(c)}${disperserStr(s.disperser)}+MIR"
```

Comment:
This seems fine according to the task. I wonder if this has any effect on the keywords
@jluhrs may know

Reply:
The keyword value is read back from the instrument. This may change it, but I don't think that would be a problem.